### DISCUSSION TITLE
Fixing plugins

### DISCUSSION THREAD

**Discussion by andyburris** [andyburris/logseq-arc-theme#20] [Announcements]
**TLDR**: Use the Discussions to submit CSS snippets that fix plugins-- put the plugin's name as the title and use the template at the bottom to comment with the snippet.

### Reasoning
I realize that this theme will most likely break some plugins' CSS. However, I've decided not to officially support plugin CSS in the theme for a number of reasons.

1. **Time**
I haven't had a ton of time generally to work on this theme, and want to use the time I do have to work on the core Logseq features and update changes.
2. **Gradual breakage**
Even if I were to add support for a plugin (or more likely as has happened, had someone give me a snippet to support a plugin), changes in the theme, the Logseq styling, or the plugin's styling all could break support. Since I don't use most of the possible plugins, I wouldn't recognize that it had broken, and the theme would be constantly out of date.

For those reasons, I've decided to open the GitHub Discussions page so that people who do use broken plugins can submit snippets of CSS to fix them. 

### Posting
 **For each plugin, you can submit a new discussion with the plugin name as the title.** Since snippets might break with theme, plugin, or Logseq updates, you can post updated snippets in each thread as new comments. (Also, if you have improvements to submit on someone else's snippet, keep it in the same thread.) Please try to use the following format for comments, so people can easily browse:

**Tested with**
Logseq \<version\> | Plugin \<version\> | Theme \<version\>

**Snippet**
```css
/* snippet css code */
```

**Notes**
*Improvement, updates, etc. that you have made. You can remove this section if there are no notes*
